shopify_python/git_utils.py

```diff
@@ -95,3 +96,15 @@ def autopep_files(files, max_line_length):
                               select={'W', 'E'},
                               verbose=0)
     autopep8.fix_multiple_files(files, options, sys.stdout)
+
+
+def pylint_files(files, **kwargs):
+    # type: (typing.List[str], **str) -> typing.Iterable[str]
+    file_string = " ".join(files)
+    args = " ".join(["--{}={}".format(key, value) for key, value in kwargs.items()])
+    stdout, _ = lint.py_run("{} {}".format(file_string, args), return_std=True)
```

Comment:
Really good question. It's led me down a path of not using `epylint` at all, and calling `pylint.lint.Run` myself. If we do that, then passing `jobs=0` sets `jobs` to the number of available cores. It then uses the `multiprocessing` module to parallelize the execution.

Not ready for that yet, I'll continue down this path tomorrow.